hearth/lib/hearth/validator.rb

```diff
@@ -17,10 +20,33 @@ def self.validate_types!(value, *types, context:)
             "[#{types.map(&:to_s).join(', ')}], got #{value.class}."
     end
 
-    # Validate a value is present and not empty.
-    # @raise [ArgumentError] Raises when the value is nil or empty.
+    # Validate a value is present and not nil.
+    # @param value [Object] The value to validate.
+    # @raise [ArgumentError] Raises when the value is nil.
     def self.validate_required!(value, context:)
```

Comment:
Just missing comment about second argument: `# @param context [String] The context of the value being validated.`